app/models/conversation.rb

```diff
@@ -30,7 +30,14 @@ class Conversation < ApplicationRecord
   def self.grouped_by_increasing_time_interval_for_user(user)
     nav_conversations = user.conversations.ordered
 
-    keys = ["Today", "Yesterday", "This Week", "This Month", "Last Month", "Older"]
+    keys = [
+      I18n.t('models.conversation.groupings.today'),
+      I18n.t('models.conversation.groupings.yesterday'),
+      I18n.t('models.conversation.groupings.this_week'),
+      I18n.t('models.conversation.groupings.this_month'),
+      I18n.t('models.conversation.groupings.last_month'),
+      I18n.t('models.conversation.groupings.older'),
+    ]
```

Comment:
I think this is a good example of a view consideration sneaking it's way into the model. It'd be really nice if we could push this up to a partial somehow. 

One slightly hacky way to do it would be to turn these keys into symbols rather than strings—or even better, we turn these into the strings that represent the i18n name! So this array would be: `keys = [ ".today", ".yesterday" ...]` and then the view which iterates over these keys (somewhere we must have `.each do |header, _|` we do display it with `<%= t header %>`

I got this hack by realizing that views support [lazy lookups of keys](https://guides.rubyonrails.org/i18n.html#lazy-lookup)

Reply:
Yeah, I hear you. I would put any changes of the logic into a separate PR. I cannot add issues, so maybe it's something you could do so that it doesn't get forgotten :-)

Reply:
Done: https://github.com/AllYourBot/hostedgpt/issues/439